frontend/public/components/catalog/catalog-page.jsx

```diff
@@ -25,6 +25,18 @@ import { ClusterServiceClassModel, ClusterServiceVersionModel } from '../../mode
 import { providedAPIsFor, referenceForProvidedAPI } from '../operator-lifecycle-manager';
 import * as operatorLogo from '../../imgs/operator.svg';
 
+export const skeleton = <div className="loading-skeleton--catalog">
```

Comment:
Perhaps this should be named `SkeletonCatalog` since there are other skeletons?

Reply:
small nit on the suggested name, we should avoid capital in the first letter since it's not actually a component, just a dom node

Reply:
changed to `skeletonCatalog`